lib/jdl/converters/parsed-jdl-to-jdl-object/parsed-jdl-to-jdl-object-converter.ts

```diff
@@ -159,7 +159,7 @@ function convertAnnotationsToOptions(
   const result = {};
   annotations.forEach(annotation => {
     const annotationName = lowerFirst(annotation.optionName);
-    const value = annotation.optionValue ? annotation.optionValue : true;
+    const value = annotation.optionValue !== undefined ? annotation.optionValue : true;
```

Comment:
I think we can use nullish coalescing

```suggestion
    const value = annotation.optionValue ?? true;
```

Reply:
changed, thanks